bin/circle/install-circle-predependencies.sh

```diff
@@ -1,23 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
Surprise!  We were no longer calling this file anywhere, so we've been using yarn 0.27.5 in our Circle builds, probably since the Circle 2.0 upgrade.  This probably accounts for some of the weird yarn.lock issues we've run into.

![image](https://user-images.githubusercontent.com/1615761/39335089-69f9fa4e-4966-11e8-917b-e51b3d1017e2.png)